### PR TITLE
Pr/unused files

### PR DESCRIPTION
**Pull request by uli42**
Drop a lot of files, this is related to #256. It removes the files we agreed can go. 

Also drops DoLoadableServer support.

### CODE REVIEW

File: nx-X11/programs/Xserver/dix/ffs.c [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-*/
-
-int
-ffs(int i)
-{
-    int j;
-    for (j = 1; (i & 1) == 0; j++)
-	i >>= 1;
-    return j;
-}
```
Comment (sunweaver) [inline nx-X11/programs/Xserver/dix/ffs.c:36]: Skip this commit for now in this PR. This affects the Win32 support discussion. This file should be removed, once we drop Win32 support.

**Review by sunweaver — CHANGES_REQUESTED** [review]
Please keep ffs.c (dix/) for now. The rest looks good.

**Comment by sunweaver** [comment]
@uli42: please rebase #267's PR branch against latest 3.6.x, as well...

File: nx-X11/programs/Xserver/dix/ffs.c (hunk 1 above)
Comment (uli42) [reply nx-X11/programs/Xserver/dix/ffs.c:36]: done

**Comment by sunweaver** [comment]
@uli42: You re-pushed last night, but the branch still had one conflict. Please rebase against latest 3.6.x again and make sure the PR branch for #267 can be merged cleanly on top of 3.6.x.

**Comment by uli42** [comment]
Unfortunately this rebase is not straight forward, I will need some time for it. Probably not today, sorry.

**Comment by sunweaver** [comment]
@uli42: I am currently rebasing this branch, so you can fetch my rebased branch and force-push to your PR branch again. Sorry for the mess.

**Comment by sunweaver** [comment]
@uli42: obtain my rebased branch with these commands:

```
git remote add gh-sunweaver git@github.com:sunweaver/nx-libs.git
git fetch gh-sunweaver pr/unused_files:sunweaver-pr/unused_files
git branch -m pr/unused_files pr/unused_files.bak
git branch -m sunweaver-pr/unused_files pr/unused_files
```

and then (after checking), force-push to the PR branch again:

```
git push --force <github-uli42-remote> pr/unused_files:pr/unused_files
```

Thanks+Greets,
Mike

**Comment by uli42** [comment]
done.

**Review by sunweaver — APPROVED** [review]

**Review by sunweaver — CHANGES_REQUESTED** [review]
@uli42: I have disapproved this PR again, as I get FTBFS at various locations.

Also, I don't think dropping damage.c is a really good idea, because it gets included in Xserver/hw/nxagent/NXdamage.c (yes, the .c file gets included).

I can continue looking at this PR and will investigate later today/tomorrow.

**Comment by sunweaver** [comment]
Also, you have moved damage.h to Xserver/include. In X.org, this file is still in miext/damage/... Moving it back there.

**Comment by sunweaver** [comment]
I have fixed the FTBFS issues and pushed to pr/unused_files on https://github.com/sunweaver/nx-libs again. Please check once more and force-push to this PR's update branch. Thanks.

**Comment by uli42** [comment]
Sorry, I missed that damage.c include somehow. I am ok with that change.

I am running out of time for now, so you can merge that from your
tree, I don't see why you'd need to use my PR.

One thing I noticed, though: commit
99b54bceaa3a6fc3d2133e08ff015c594fdff013 contains a now historical
conflicts message. That one should vanish.

**Comment by sunweaver** [comment]
Obsoleted by 1d1d1fc2199672c722dd75cefe8389268bfbbad3. Closing.

**Pull request closed without merging**